README.md

```diff
@@ -94,7 +111,14 @@ Your Answer:
 What are the differences between calling and referencing a function? Please provide examples of each.
 
 ```text
+for var exampleFunction = function() {
+  console.log("Question 8");
+}
+Calling a function: exampleFunction();
+
+Referencing a function -  exampleFunction;
 
+Calling a function executes the function immediately. Referencing a function stores it and allows it to be executed at a later time.
 ```
```

Comment:
Referencing doesn't store the function. Instead, it returns the function definition (which assumes it was previously defined).